### PR TITLE
Added new interfaces

### PR DESCRIPTION
**Pull request by Alex99y**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -261,7 +275,7 @@ class MyAlgoConnect {
 
 			const res = await this.bridge.sendMessage(
 				this.currentConnectPopup,
-				{ method: "unlock", params: { shouldSelectOneAccount: settings.shouldSelectOneAccount } },
+				{ method: "unlock", params: Object.assign(settings, { disableLedgerNano: this.disableLedgerNano }) },
```
Comment (Ignacio-87) [inline lib/main.js:278]: We need to start to use _.deepCopy fron loadash library instead  this function, this is a shallow copy

**Review by Ignacio-87 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # MyAlgo Connect Change log
 
+## [1.1.2] - 10-01-2022
+
+### Added
+
+- Added new constructor param `disableLedgerNano`
+- Added new two param to connect method `openManager` and `shouldSelectOneAccount`
```
[the remark this answers is not in the document]
Comment (Alex99y) [reply CHANGELOG.md]: done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # MyAlgo Connect Change log
 
+## [1.1.2] - 10-01-2022
+
+### Added
+
+- Added new constructor param `disableLedgerNano`
```
[the remark this answers is not in the document]
Comment (Alex99y) [reply CHANGELOG.md]: done.

File: index.d.ts [hunk 4]
```diff
@@ -171,7 +173,7 @@ export default class MyAlgoConnect {
 	/**
 	 * @async
 	 * @description Receives user's accounts from MyAlgo.
-	 * @param {ConnectionSettings} Connection settings
+	 * @param {ConnectionSettings} [settings] settings
```
[the remark this answers is not in the document]
Comment (Alex99y) [reply index.d.ts]: done.

File: lib/main.js [hunk 5]
```diff
@@ -23,8 +23,17 @@ const bridge = new Messaging();
   * @type {object}
   * @property {string} [bridgeUrl] Override wallet.myalgo.com default frame url.
   * @property {number} [timeout] Number of msec to wait the popup response, default value: 1600000 msec.
+  * @property {boolean} [disableLedgerNano] Disable Ledger Nano accounts, it will return only mnemonic accounts
```
[the remark this answers is not in the document]
Comment (Alex99y) [reply lib/main.js]: improve it and done.

File: lib/main.js (hunk 1 above)
Comment (Alex99y) [reply lib/main.js:278]: We don't need a deep copy, only merging both objects.

**Review by Ignacio-87 — APPROVED** [review]

**Pull request merged**